src/main/mondrian/rolap/SqlTupleReader.java

```diff
@@ -90,6 +90,16 @@
     private int missedMemberCount;
     private static final String UNION = "union";
     private int emptySets = 0;
+    // allow hints by default
+    private boolean allowHints = true;
+
+    public boolean isAllowHints() {
+        return allowHints;
+    }
+
+    public void setAllowHints(boolean allowHints) {
+        this.allowHints = allowHints;
+    }
 
```

Comment:
@mkambol, since ```SqlTupleReader``` is a public class and it has a public setter, why shouldn't it has a public getter even though is it not used?